lib/pure/sugar.nim

```diff
@@ -201,7 +201,7 @@ macro capture*(locals: varargs[typed], body: untyped): untyped {.since: (1, 1).}
   ##     for j in 7..9:
   ##       if i * j == 42:
   ##         capture i, j:
-  ##           myClosure = proc () = echo fmt"{i} * {j} = 42"
+  ##           myClosure = proc () = echo fmt"{i} * {j} == 42"
   ##   myClosure() # output: 6 * 7 == 42
```

Comment:
@Araq sorry if I was unclear, but this comment is incorrect then and it's been here before :)